common/src/main/java/com/mx/common/connect/ConnectException.java

```diff
@@ -1,6 +1,8 @@
-package com.mx.common.exception;
+package com.mx.common.connect;
 
+import com.mx.common.accessors.AccessorException;
```

Comment:
Do we still want this to extend `AccessorException`, or were we going to change it to inherit from `PathRequestException` directly?

Reply:
Fixed